stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -12,7 +12,8 @@
 
 /// A fixed-width integer that is twice the size of its base type.
 public struct DoubleWidth<Base : FixedWidthInteger> :
-  FixedWidthInteger, _ExpressibleByBuiltinIntegerLiteral {
+  FixedWidthInteger, _ExpressibleByBuiltinIntegerLiteral
+  where Base.Words : Collection, Base.Magnitude.Words : Collection {
```

Comment:
Is this necessary?

If so, that may be a pretty good sign that `Words` should conform to `Collection` once it becomes possible to do so; if not, then it should be possible to implement `DoubleWidth.Words` without this restriction.